controller/checkAccess/checkAccess.js

```diff
@@ -7,6 +7,9 @@
 /* eslint-disable valid-jsdoc */
 /* eslint-disable max-depth */
 /* eslint-disable max-lines */
+
+const _ = require('lodash');
```

Comment:
we've never used lodash before, but I'm happy to start. What are the main things you use it for?

Reply:
Ah, sorry, I didn't see your message.
I think it's a really handy library. There's a lot of helpers that make working with collections especially great, be they arrays or objects. It's to be used in a functional way, which I think is nice as it's more declarative / makes your intent more clear.

You may read posts about people saying that it's now unnecessary and adds bloat but it comes mostly from frontend developers concerned with the size of their final bundle working in an E6 context. For node and js < ES6 I think it makes sense to use it.